src/packages/cli/src/__tests__/dotenv-3-conflict.test.ts

```diff
@@ -0,0 +1,13 @@
+import { consoleContext, Context } from './__helpers__/context'
+
+const ctx = Context.new().add(consoleContext()).assemble()
+
+it('should throw error', async () => {
+  ctx.fixture('dotenv-3-conflict')
```

Comment:
Why do we throw an error when there is a duplicate env var?

Default behavior of dotenv is to take the env var from system and ignore the one in the `.env` file.

Are we trying to read and load both the root and prisma folder `.env` file?

Reply:
https://github.com/prisma/prisma/issues/3720#issuecomment-704246239

Reply:
So this
```
Proposed Solution

The CLI will look for .env in the project root in addition to the prisma/ folder. During prisma generate, we'll generate a relative path link from the generated client to .env.

Current users may have both a .env and a prisma/.env. We'll continue to allow both files for now. If there's a required environment variable that's present in both, we'll throw an error when you generate. For example, if DATABASE_URL is defined in both .env and prisma/.env, we'll throw an error when you run prisma generate.
```

but what should the user do? Remove it from root .env or prisma folder .env? We could help the users decide what to do here I think.

Reply:
Yeah, I would say remove the `.env` file in the Prisma folder and move all vars to root. As that is the direction we are going

Reply:
In that case, they probably want to remove it from the prisma folder to unify their .env

Reply:
Definitely makes sense to add that to the error message, good point

Reply:
Yeah Matt and I came up with that, we just want to make sure, that it's strict in the beginning and fail loud, as having wrong env vars / wrong credentials can really go wrong.
We can still later lift that restriction.